controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -373,6 +400,18 @@ private File determineMainFile(final String rawName, final String name) {
         throw ControllerLogger.ROOT_LOGGER.mainFileNotFound(name != null ? name : rawName, configurationDir);
     }
 
+    private String translateFileAlias(String alias) {
+        final int pos = rawFileName.lastIndexOf('.');
+        final String base = pos != -1 ? rawFileName.substring(0, pos) : rawFileName;
```

Comment:
If you make the final name dependent on the `rawFileName` you would need to clarify it on the proposal analysis doc.
As far as I understand, the `rawFileName` is the name default name for configuration files. This default name can be modified by `jboss.server.default.config`. If the base of the translate alias depends on that, we need to describe this dependency on the analysis doc file to don't add any unadvertised behaviour.

Reply:
I'll clarify in the proposal then, making it only work with files specifically named "standalone-" seems needlessly limiting.